File: 1020/1020.java
// bee 1020 - Idade em Dias
// Alonso Martins
// 18/04/2024
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int a, m, d = scanner.nextInt();
        
        a = d / 365;
        d %= 365;
        m = d / 30;
        d %= 30;

        System.out.println(a + " ano(s)");
        System.out.println(m + " mes(es)");
        System.out.println(d + " dia(s)");
        
        scanner.close();
    }
}

File: 1020/1020.py
# bee 1020 - Idade em Dias
# Alonso Martins
# 20/07/2023
idade = int(input())
print(f"{idade//365} ano(s)")
print(f"{(idade%365)//30} mes(es)")
print(f"{(idade%365)%30} dia(s)")
